samples/grpc_server/main.go

```diff
@@ -247,7 +260,9 @@ func main() {
 	}
 	grpcServer := grpc.NewServer(opts...)
 	RegisterRouteGuideServer(grpcServer, newServer())
+	RegisterTheSecondServiceServer(grpcServer, newServer2())
 	reflection.Register(grpcServer)
+	fmt.Printf("gRPC server is listining on the port: %d\n", *port)
```

Comment:
```suggestion
	fmt.Printf("gRPC server is listening on the port: %d\n", *port)
```

Reply:
Skipping this because it's part of a commit that we expect to drop before going to merge.